draft-ietf-lamps-kyber-certificates.md

```diff
@@ -250,6 +250,15 @@ encoded using the textual encoding defined in {{?RFC7468}}.
 
 # Private Key Format
 
+An ML-KEM private key is encoded by storing its 64-octet seed in
+the privateKey field as follows.
```

Comment:
I agree with using the 64-byte seed, but technically, a FIPS 203-compliant ML-KEM private key is / can be either a 64-byte seed or the fully expanded decapsulation key format; in this phrase, what is the meaning of 'is'? The ML-KEM in use in this document?  Or the ML-KEM in FIPS 203? (Making this harmonious with the next paragraph would help)

Reply:
Strictly speaking this sentence could be removed from the standard. It's here as a stepping stone to the precise definition. We could add a marker like "essentially" or "in short".

Reply:
Cool, went with 'in short': https://github.com/lamps-wg/kyber-certificates/pull/68